gluonnlp/_constants.py

```diff
@@ -36,6 +38,9 @@
 
 LARGE_NEGATIVE_FLOAT = -LARGE_POSITIVE_FLOAT
 
+INT_TYPES = (int, np.int8, np.int, np.int16, np.int32, np.int64,
+             np.uint8, np.uint16, np.uint32, np.uint64)
```

Comment:
Do we need all these options? Also, we can probably reuse what has been declared in mxnet base. https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/base.py#L40

Reply:
You are right. We may not really need something like int8.